src/switch_ivr_play_say.c

```diff
@@ -1280,6 +1280,13 @@ SWITCH_DECLARE(switch_status_t) switch_ivr_play_file(switch_core_session_t *sess
 
 
 	for (cur = 0; switch_channel_ready(channel) && !done && cur < argc; cur++) {
+		const int sp_fadeLen = 32;
+		const int sp_cut_src_rng = 64;
+		short sp_ovrlap[32];
+		short sp_has_overlap = 0;
+		int sp_prev_idx = 0;
+		short sp_prev[SWITCH_RECOMMENDED_BUFFER_SIZE];
```

Comment:
maybe initialize to 0? 
`short sp_prev[SWITCH_RECOMMENDED_BUFFER_SIZE] = { 0 };`